Symfony/CS/Fixer/Contrib/NoBlankLinesBeforeNamespaceFixer.php

```diff
@@ -26,7 +26,7 @@ public function fix(\SplFileInfo $file, $content)
     {
         $tokens = Tokens::fromCode($content);
 
-        foreach (array_keys($tokens->findGivenKind(T_NAMESPACE)) as $index) {
+        foreach ($tokens->findGivenKind(T_NAMESPACE) as $index => $token) {
```

Comment:
@keradus We could have avoided this if we just returned an iterator from findGivenKind in 2.0?